src/app/form-generator/components/node-field/node-field.component.html

```diff
@@ -1,10 +1,58 @@
-<label>{{ api.field.label || api.field.name }}</label>
-<div class="uuid-form">
+<gtx-contents-list-item>
+    <div class="title"
+         *ngIf="api.field.label || api.field.name">
+        <label>{{ api.field.label || api.field.name }}</label>
+    </div>
+    <div class="item-primary button select-item-button"
+         *ngIf="!userValue">
+        <gtx-button flat="true"
+                    (click)="selectNode()">
+            <icon left>search</icon>
+            Select Node Reference
+        </gtx-button>
+    </div>
+    <div *ngIf="userValue"
+         class="icon">
+
+        <mesh-thumbnail [nodeUuid]="userValue"
+                        *ngIf="!isContainer">
+        </mesh-thumbnail>
+
+        <icon *ngIf="isContainer">folder</icon>
+    </div>
+    <div class="item-primary"
+         *ngIf="userValue">
+        <ul>
+            <li class="schemaName">
+                <span class="innerSchemaName">
+                    <icon>view_quilt</icon>
+                    <div>{{ schemaName }}</div>
+                </span>
+            </li>
+            <li>
+                <a class="nodeName"
+                   [routerLink]="routerLink"
+                   (click)="editNode()">{{ displayName }}</a>
+            </li>
+            <li>
+                <label class="path">{{ breadcrumbPath }}</label>
```

Comment:
Don't use label here. Create a div and assign it the `path` class. In that class you can set font-size and color